lib/postgrex/replication.ex

```diff
@@ -286,6 +259,96 @@ defmodule Postgrex.Replication do
     Connection.start_link(__MODULE__, {module, arg, connection_opts}, server_opts)
   end
 
+  @doc """
+  Creates a logical replication slot with the given name and output plugin.
+  By default, PostgreSQL includes the `pgoutput` plugin.
```

Comment:
Keep the first paragraph short as it is used as a summary by exdoc. :)

```suggestion

  By default, PostgreSQL includes the `pgoutput` plugin.
```